GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js

```diff
@@ -5,68 +5,81 @@
-
-    /** @type {PIXI.SystemRenderer} */
-    this._pixiRenderer = null;
-    this._canvasWidth = 0; // Current width of the canvas (might be scaled down/up compared to renderer)
-    this._canvasHeight = 0; // Current height of the canvas (might be scaled down/up compared to renderer)
-    this._keepRatio = true;
-    this._marginLeft = this._marginTop = this._marginRight = this._marginBottom = 0;
-}
+gdjs.RuntimeGamePixiRenderer = function(game, forceFullscreen) {
+  this._game = game;
+
+  this._isFullPage = true; //Used to track if the canvas is displayed on the full page.
+  this._isFullscreen = false; //Used to track if the window is displayed as fullscreen (see setFullscreen method).
+  this._forceFullscreen = forceFullscreen; //If set to true, the canvas will always be displayed as fullscreen, even if _isFullscreen == false.
+
+  /** @type {PIXI.SystemRenderer} */
+  this._pixiRenderer = null;
+  this._canvasWidth = 0; // Current width of the canvas (might be scaled down/up compared to renderer)
+  this._canvasHeight = 0; // Current height of the canvas (might be scaled down/up compared to renderer)
+  this._keepRatio = true;
+  this._marginLeft = this._marginTop = this._marginRight = this._marginBottom = 0;
+};
 
 gdjs.RuntimeGameRenderer = gdjs.RuntimeGamePixiRenderer; //Register the class to let the engine use it.
 
 /**
  * Create a standard canvas inside canvasArea.
  *
  */
-gdjs.RuntimeGamePixiRenderer.prototype.createStandardCanvas = function(parentElement) {
-    //This prevents flickering on some mobile devices
-    PIXI.glCore.VertexArrayObject.FORCE_NATIVE = true;
-
-    //Create the renderer and setup the rendering area
-    //"preserveDrawingBuffer: true" is needed to avoid flickering and background issues on some mobile phones (see #585 #572 #566 #463)
-    this._pixiRenderer = PIXI.autoDetectRenderer(this._game.getGameResolutionWidth(), this._game.getGameResolutionHeight(), {
-        preserveDrawingBuffer: true,
-    } );
-    parentElement.appendChild(this._pixiRenderer.view); // add the renderer view element to the DOM
-    this._pixiRenderer.view.style["position"] = "absolute";
-    this._pixiRenderer.view.tabindex = "1"; //Ensure that the canvas has the focus.
-    this._resizeCanvas();
-
-    // Handle scale mode
-    if (this._game.getScaleMode() === "nearest") {
-        this._pixiRenderer.view.style["image-rendering"] = "-moz-crisp-edges";
-        this._pixiRenderer.view.style["image-rendering"] = "-webkit-optimize-contrast";
-        this._pixiRenderer.view.style["image-rendering"] = "-webkit-crisp-edges";
-        this._pixiRenderer.view.style["image-rendering"] = "pixelated";
+gdjs.RuntimeGamePixiRenderer.prototype.createStandardCanvas = function(
+  parentElement
+) {
+  //This prevents flickering on some mobile devices
+  PIXI.glCore.VertexArrayObject.FORCE_NATIVE = true;
+
+  if (this._game.getScaleMode() === 'nearest') {
```

Comment:
@Bouh, I think we also remove these lines there? To keep things consistent?

Reply:
I keep   PIXI.glCore.VertexArrayObject.FORCE_NATIVE = true;   ?

Selection on review on Github are a bit bad i didn't understand which lines you means..

Reply:
These:
![image](https://user-images.githubusercontent.com/1280130/74615357-e49fd880-5117-11ea-8646-8217b311482a.png)

![image](https://user-images.githubusercontent.com/1280130/74615354-dce03400-5117-11ea-83f0-e2d802772338.png)

Reply:
Ok done!